pkg/remote/crd_upgrade_test.go

```diff
@@ -0,0 +1,100 @@
+package remote_test
+
+import (
+	"testing"
+
+	"github.com/kyma-project/lifecycle-manager/api/v1beta2"
+	"github.com/kyma-project/lifecycle-manager/pkg/remote"
+	"github.com/stretchr/testify/assert"
+	v1extensions "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+//nolint:funlen
+func TestShouldPatchRemoteCRD(t *testing.T) {
+	t.Parallel()
+	type args struct {
+		runtimeCrdGeneration      int64
+		kcpCrdGeneration          int64
+		kymaKcpCrdAnnotationValue string
+		kymaSkrCrdAnnotationValue string
+		err                       error
```

Comment:
what was the usage of this err here? If have a specific purpose, you need to test this err, but in your test case, all given value is nil.